src/tuntap/tun.go

```diff
@@ -177,6 +180,11 @@ func (tun *TunAdapter) _start() error {
 	return nil
 }
 
+// IsStarted returns true if the module has been started.
+func (tun *TunAdapter) IsStarted() bool {
+	return tun.isOpen
```

Comment:
Accessing `tun.isOpen` isn't thread safe, and probably needs to be in a `phony.Block` to run in the actor.